samples/net/zperf/src/zperf_tcp_receiver.c

```diff
@@ -166,14 +183,29 @@ void zperf_tcp_receiver_init(const struct shell *shell, int port)
 			return;
 		}
 
-		ret = zperf_get_ipv6_addr(shell, MY_IP6ADDR, MY_PREFIX_LEN_STR,
-					  &in6_addr_my->sin6_addr);
-		if (ret < 0) {
-			shell_fprintf(shell, SHELL_WARNING,
-				      "Unable to set IPv6\n");
-			return;
+		if (MY_IP6ADDR) {
```

Comment:
same here

Reply:
Same reason becase of compile error if MY_IP6ADDR is NULL.

Reply:
> same here

Same reason becase of compile error if MY_IP6ADDR is NULL.